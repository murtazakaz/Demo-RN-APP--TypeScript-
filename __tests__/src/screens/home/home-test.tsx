import React from 'react';
import renderer from 'react-test-renderer';

import Home from '../../../../src/screens/home/home';

it('renders correctly', () => {
  renderer.create(<Home />);
});

it('renders correctly home screen even with empty', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
