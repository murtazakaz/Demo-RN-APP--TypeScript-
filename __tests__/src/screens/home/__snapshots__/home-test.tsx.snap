// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly home screen even with empty 1`] = `
<View
  accessibilityIgnoresInvertColors={true}
  style={
    Object {
      "height": "100%",
      "width": 750,
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/images/homeBG.png",
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": "100%",
          "width": 750,
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "paddingLeft": 56,
        "paddingTop": 310,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 120,
          "fontWeight": "700",
        }
      }
    >
      Hello
    </Text>
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 24,
          "fontWeight": "400",
        }
      }
    >
       0 Livestream  - 0 Member
    </Text>
  </View>
</View>
`;
